src/uct/tcp/tcp_sockcm.h

```diff
@@ -7,30 +7,39 @@
 #include <uct/base/uct_cm.h>
 
 
+typedef struct uct_tcp_sockcm_ep   uct_tcp_sockcm_ep_t;
```

Comment:
why don't define it here?

Reply:
`uct_tcp_sa_arg_t` depends on this structure, but this structure defined after
@alinask right?

Reply:
i think it's better to have the definition itself in the ep header file

Reply:
@dmitrygx right.

Reply:
> > why don't define it here?
> 
> i think it's better to have the definition itself in the ep header file

Yes, My question was why the structure is defined in source but `uct_tcp_sockcm_ep_t` here? Why don't define complete type here?